concordium-std/tests/derives.rs

```diff
@@ -0,0 +1,13 @@
+#[test]
```

Comment:
Please document the intent of this test as a module doc comment, i.e., //!.

Reply:
Added